src/v1.js

```diff
@@ -48,7 +48,7 @@ function v1(options, buf, offset) {
   // (1582-10-15 00:00).  JSNumbers aren't precise enough for this, so
   // time is handled internally as 'msecs' (integer milliseconds) and 'nsecs'
   // (100-nanoseconds offset from msecs) since unix epoch, 1970-01-01 00:00.
-  let msecs = options.msecs !== undefined ? options.msecs : Date.now();
+  let msecs = options.msecs !== undefined ? options.msecs : new Date().getTime();
```

Comment:
Just out of curiosity, why was `new Date().getTime()` preferred over `Date.now()`? ☺️

Reply:
Because with `Date.now()` the test `docs:diff` fails… >_<

Reply:
For updating the docs please edit `README_js.md` and then `npm run docs`, then commit the changes to both `README_js.md` and `README.md`.

Reply:
@ctavan with `Date.now()` different values are always generated. Therefore it is impossible to pass the test `docs:diff`. Try it yourself. I spent an hour to understand what was happening )